apps/server/src/solana-contract/user/user.service.ts

```
@@ -0,0 +1,142 @@
+			// Getting prepared the transaction
+			const preparedTransaction =
+				await this.server.prepareTransaction(transaction)
+
+			// Getting signed the transaction
+			preparedTransaction.sign(this.sourceKeypair)
+
+			// Sending the transaction
+			const response = await this.server.sendTransaction(preparedTransaction)
+
+			if (response.status === 'PENDING') {
+				let getResponse
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)